tests/behaviour/qbwriter.feature

```diff
@@ -646,3 +647,79 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     
     <file:/tmp/qb-id-10002.csv#csvcubed-build-activity> a prov:Activity;
     """
+
+Scenario: A pivoted shape cube should be produced as the output for the pivoted shape inputs
+  Given a pivoted shape cube with identifier "qb-id-10003" named "Pivoted Shape Cube"
+  When the cube is serialised to CSV-W
+  Then the file at "qb-id-10003.csv" should exist
+  And the file at "qb-id-10003.csv-metadata.json" should exist
+  And csvlint validation of all CSV-Ws should succeed
+  And csv2rdf on all CSV-Ws should succeed
+  And the RDF should contain
+  """
+    @prefix cube: <http://purl.org/linked-data/cube#> .
+    @prefix attribute: <file:/tmp/qb-id-10003.csv#attribute/> .
+    @prefix measure: <file:/tmp/qb-id-10003.csv#measure/> .
+    @prefix dimension: <file:/tmp/qb-id-10003.csv#dimension/> .
+    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+    
+    <file:/tmp/qb-id-10003.csv#slice/a> a cube:Slice ;
+        dimension:some-dimension <file:/tmp/some-dimension.csv#a> ;
+        cube:Observation <file:/tmp/qb-id-10003.csv#obs/a@some-measure> ;
+        cube:sliceStructure <file:/tmp/qb-id-10003.csv#slice/cross-measures> .
+    
+    <file:/tmp/qb-id-10003.csv#slice/b> a cube:Slice ;
+        dimension:some-dimension <file:/tmp/some-dimension.csv#b> ;
+        cube:Observation <file:/tmp/qb-id-10003.csv#obs/b@some-measure> ;
+        cube:sliceStructure <file:/tmp/qb-id-10003.csv#slice/cross-measures> .
+    
+    <file:/tmp/qb-id-10003.csv#slice/c> a cube:Slice ;
+        dimension:some-dimension <file:/tmp/some-dimension.csv#c> ;
+        cube:Observation <file:/tmp/qb-id-10003.csv#obs/c@some-measure> ;
+        cube:sliceStructure <file:/tmp/qb-id-10003.csv#slice/cross-measures> .
+    
+    <file:/tmp/qb-id-10003.csv#obs/a@some-measure> a cube:Observation ;
+        attribute:some-attribute <file:/tmp/attr-a> ;
+        dimension:some-dimension <file:/tmp/some-dimension.csv#a> ;
+        measure:some-measure 1.0,
```

Comment:
This isn't right, there shouldn't be two values for the same measure. This may be symptomatic of a bug.